lms/djangoapps/courseware/views/views.py

```diff
@@ -1685,9 +1682,7 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
             'staff_access': bool(request.user.has_perm(VIEW_XQA_INTERFACE, course)),
             'xqa_server': settings.FEATURES.get('XQA_SERVER', 'http://your_xqa_server.com'),
             'display_reset_dates_banner': display_reset_dates_banner,
-            'reset_deadlines_url': reset_deadlines_url,
-            'reset_deadlines_redirect_url_base': reset_deadlines_redirect_url_base,
-            'reset_deadlines_redirect_url_id_dict': {'course_id': str(course.id), 'usage_key_string': usage_key_string}
+            'web_app_course_url': reverse(COURSE_HOME_VIEW_NAME, args=[course.id])
```

Comment:
this is the problem view, where we currently see the banner+button showing up, correct?

Reply:
yep, and so now, we put the course home url in the context so that the link can send the user there, and we remove the context with the info for the button itself.